### ISSUE TITLE
Workaround does not work due to changed variable names

### ISSUE THREAD

**Issue by jojo2357** [helgatheviking/Nav-Menu-Roles#70]
I tried to get the plugin to work with 2024 (WP default) and pasted 

`do_action( 'wp_nav_menu_item_custom_fields', $item_id, $item, $depth, $args, $id );`

in `wp-admin/includes/class-walker-nav-menu-checklist.php` and noticed that `$item_id` `$item` `$id` are not in the scope of `start_el` with a comment that two of those names were changed circa wp `5.9.0`

I couldn't get the menu to show up in wp-admin menu to config. I have used with 2021 theme some time ago and it worked great. Thanks btw :)

**Comment by helgatheviking** [comment]
You shouldn't need a workaround for twenty twenty four theme. wp_nav_menu_item_custom_fields is now in core. I'm not sure I understand what is happening, can you clarify?

**Comment by jojo2357** [comment]
When I install the plugin I can't find the settings where I have seen them in prior versions. Where are they? I expect them under `Appearance > Menus` but I see no such item.

**Comment by helgatheviking** [comment]
Yes, they should be on each menu item you see when editing a menu via appearance > menu. 

If you aren't seeing them, you likely have a conflict and I would suggest switching themes and disabling plugins one at a time.

**Comment by helgatheviking** [comment]
It's just now occurred to me that Appearance > Menus isn't accessible at all in the admin.... Is this the problem? Currently there's no support for the nav menus in the block editor, but I think it might be possible.

**Comment by jojo2357** [comment]
Recently I had a reason to spin up a base wp install and yes, as you mention it doesnt show up in the menu

**Comment by helgatheviking** [comment]
Thank you for confirming. I hadn't realized the menu had gone away. I will have to think about how to handle that. Technically I think the block editor is customizable enough, but I'm only mid in block development so that might a lot of work.
